app/src/main/java/org/simlar/https/HttpsPost.java

```diff
@@ -124,16 +125,31 @@ public static InputStream post(final String urlPath, final Map<String, String> p
 		return null;
 	}
 
+	private static OutputStream getOutputStream(final HttpsURLConnection connection)
+	{
+		if (connection == null) {
+			return null;
+		}
+
+		try {
+			return connection.getOutputStream();
+		} catch (final IOException e) {
+			Lg.ex(e, "IOException while getting OutputStream");
+			return null;
+		}
+	}
+
 	private static InputStream postPrivate(final String urlPath, final Map<String, String> parameters)
 	{
 		final HttpsURLConnection connection = createConnection(urlPath, false);
 
-		if (connection == null) {
+		final OutputStream stream = getOutputStream(connection);
+		if (stream == null) {
 			return null;
 		}
 
+		final PrintWriter out = new PrintWriter(stream);
 		try {
-			final PrintWriter out = new PrintWriter(connection.getOutputStream());
```

Comment:
I am not sure whether moving `connection.getOutputStream()` into a separate function did make the exception handling more readable. At least under the assumption that it will never return `null` but will throw an exception instead.